massgov/mayflower
# DP-10765: Porting Table of Contents Hierarchy component.

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)

## Description
This ports the [Table of Contents Hierarchy component from patternlab](https://mayflower.digital.mass.gov/?p=organisms-table-of-contents-hierarchy) to mayflower react. The twig side of things had code for the SidebarHeading component as well, so I made it work for that. Passing either a SidebarHeading or ColoredHeading component to the heading prop will render either.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-10765)

## Steps to Test


1. Browse to Organisms -> Table of Contents: ColoredHeading (also Table of Contents: SidebarHeading) and verify the look/feel works for the port.

## Screenshots
![screencapture-localhost-6006-2018-10-12-11_25_36](https://user-images.githubusercontent.com/5257660/46878691-e665d780-ce11-11e8-95ef-1378a28a3807.png)
